src/qml/nodemodel.h

```diff
@@ -25,15 +29,24 @@ class NodeModel : public QObject
 {
     Q_OBJECT
     Q_PROPERTY(int blockTipHeight READ blockTipHeight NOTIFY blockTipHeightChanged)
+    Q_PROPERTY(int remainingSyncTime READ remainingSyncTime NOTIFY remainingTimeChanged)
     Q_PROPERTY(double verificationProgress READ verificationProgress NOTIFY verificationProgressChanged)
+    Q_PROPERTY(QVariantList blockTimeList READ blockTimeList NOTIFY blockTimeListChanged)
+    Q_PROPERTY(double currentTime READ currentTime NOTIFY currentTimeChanged)
 
 public:
     explicit NodeModel(interfaces::Node& node);
 
     int blockTipHeight() const { return m_block_tip_height; }
     void setBlockTipHeight(int new_height);
+    int remainingSyncTime() const { return m_remaining_time; }
     double verificationProgress() const { return m_verification_progress; }
     void setVerificationProgress(double new_progress);
+    QVariantList blockTimeList() const { return m_block_time_list; }
+    bool setBlockTimeList(int new_block_time);
```

Comment:
why should the nodemodel, a proxy class for our internal node, keep something like a blocklist?

Reply:
Considering your previous suggestion about using the `interfaces::chain`, I think I can create a separate .cpp file handling chain-related parameters such as blockTimeList